modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts

```diff
@@ -344,7 +348,7 @@ const toggleSingleList = (editor: Editor, parentList: HTMLElement | null, listNa
 
 const toggleList = (editor: Editor, listName: 'UL' | 'OL' | 'DL', _detail: ListDetail | null): void => {
   const parentList = Selection.getParentList(editor);
-  if (isWithinNonEditableList(editor, parentList) || hasNonEditableBlocksSelected(editor)) {
+  if (isWithinNonEditableList(editor, parentList) || isInNonEditableRoot(editor)) {
```

Comment:
Do we consider elsewhere if we're in a CEF element which is not the root? So we don't accidentally start editing CEFs.

Reply:
Yeah, it is checked one more time in `applyList` function [here](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts#L200-L202).